src/fieldset/fieldset.yaml

```diff
@@ -7,4 +7,5 @@ examples:
   data:
     legend:
       text: What is your address?
+      classes: govuk-fieldset__legend--xl
```

Comment:
Not blocking (since can be changed later) but this exposes internals (https://github.com/alphagov/govuk-frontend/issues/460)

Reply:
It's consistent with the way we do it everywhere else at the minute.